### PR TITLE
Feature/sec 47

### PR DESCRIPTION
**Pull request by gizet**

### CODE REVIEW

File: src/actions/securityActions.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        // type: GET_ERRORS,
+        // payload: err.toLocaleString()
+        //});
+    }
+};
+
+export const logout = () => async dispatch => {
+        //post token to backend
+        try {
+            //const securityHeader = {'authorization': localStorage.jwtToken};
```
Comment (CosminaIacob) [inline src/actions/securityActions.js]: Do we need this later?

File: src/components/Login.js [hunk 2]
```diff
@@ -22,16 +22,27 @@ class Login extends Component {
...
+        }
+    }
+
     // componentDidMount() {
     //     if (this.props.security) {
     //         this.props.history.push("/login");
     //     }
     // }
     //
-    // componentWillReceiveProps(nextProps) {
```
Comment (CosminaIacob) [inline src/components/Login.js]: Do we need componentDidMount commented above?

File: src/actions/securityActions.js (hunk 1 above)
Comment (gizet) [reply src/actions/securityActions.js]: am stes. am nevouie sa fac callul la redis care inca nu merge

File: src/components/Login.js (hunk 2 above)
Comment (gizet) [reply src/components/Login.js]: am sters

File: src/reducers/securityReducer.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import {SET_CURRENT_USER} from "../actions/types";
+
+const initialState = {
+    validToken: false,
+    user: {}
+};
+
+const booleanActionPayload = payload => {
+    return !!(payload)
```
[the remark this answers is not in the document]
Comment (gizet) [reply src/reducers/securityReducer.js]: nu
e practic 
  if (payload) {
    return true;
  } else {
    return false;

**Pull request merged**